stories/components/person/person.properties.stories.js

```diff
@@ -171,6 +171,17 @@ export const personPresence = () => html`
      ></mgt-person>
  `;
 
+export const myPresence = () => html`
+  <div>
+    <div>Please login...</div>
+    <mgt-login />
+  </div>
+  <div style="margin-top: 32px">
+    <div>Then change your presence in Teams to be reflected here...</div>
+    <mgt-person person-query="me" avatar-size="large" show-presence></mgt-person>
+  </div>
```

Comment:
The <mgt-login> wouldn't be required here as it's handled at the Storybook level. Just that story is perfect!

```suggestion
    <div>Change your presence in Teams to see the presence reflected here...</div>
    <mgt-person person-query="me" avatar-size="large" show-presence></mgt-person>
```